### PR TITLE
[DE] HassListAddItem

### PR DESCRIPTION
**Pull request by steffenrapp**
German translation of #1626

### CODE REVIEW

File: tests/de/todo_HassListAddItem.yaml
Comment (mib1185) [inline tests/de/todo_HassListAddItem.yaml:1]: we should also have some test sentences with "meine[r]"

**Review by mib1185 — CHANGES_REQUESTED** [review]
Hi @steffenrapp ,
good job, just one comment about missing test sentences 🙂

File: tests/de/todo_HassListAddItem.yaml
Comment (steffenrapp) [reply tests/de/todo_HassListAddItem.yaml:1]: Sure, I also added some more missing tests and also sentences 👍

**Review by mib1185 — APPROVED** [review]
LGTM, thx @steffenrapp 👍

**Review by mib1185 — CHANGES_REQUESTED** [review]
there is a missing blank line at the end of `tests/de/_fixtures.yaml`

**Comment by steffenrapp** [comment]
Thx, that missing blank line got lost when I resolved the conflicts ...

**Review by mib1185 — APPROVED** [review]

**Pull request merged**